dm/tests/dmctl_command/run.sh

```diff
@@ -81,7 +81,11 @@ function run() {
 	cp $cur/conf/source1.yaml $WORK_DIR/source1.yaml
 	cp $cur/conf/source2.yaml $WORK_DIR/source2.yaml
 	sed -i "/relay-binlog-name/i\relay-dir: $WORK_DIR/worker1/relay_log" $WORK_DIR/source1.yaml
-	dmctl_operate_source create $WORK_DIR/source1.yaml $SOURCE_ID1
+	dmctl_operate_source create $WORK_DIR/source1.yaml $SOURCE_ID1 -w $WORKER1_NAME
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"list-member --name worker1" \
+		"\"stage\": \"bound\"" 1 \
+		"\"source\": \"mysql-replica-01\"" 1
 	dmctl_operate_source create $WORK_DIR/source2.yaml $SOURCE_ID2
```

Comment:
How about add some negative test? e.g. special a bounded/no-exist worker for source2?